syntax = "proto2";


message AddReq {
    required int32 first = 1;
    required int32 second = 2;
}

message AddResp {
    required int32 result = 1;
}

message SubReq {
    required int32 first = 1;
    required int32 second = 2;
}

message SubResp {
    required int32 result = 1;
}

